paper-fab-speed-dial-action.html

```diff
@@ -43,11 +43,13 @@
 				padding: 5px 10px;
 				border-radius: 3px;
 				margin-right: 20px;
+				@apply --paper-fab-speed-dial-action-label;
```

Comment:
This seems to be not related to "allow setting of 'src' property'". Could you please split the styling related changes into a separated PR.
(There are more such cases in the PR, e.g. `paper-fab-speed-dial-action-fab`. They should all go into the styling PR.)